src/BuildScriptGenerator/DotNetCore/DotNetCoreSdkVersions.cs

```diff
@@ -11,7 +11,7 @@ public static class DotNetCoreSdkVersions
         public const string DotNetCore31SdkVersion = "3.1.415";
         public const string DotNetCore50SdkVersion = "5.0.302";
         public const string DotNet50SdkVersion = "5.0.403";
-        public const string DotNet60SdkVersion = "6.0.100";
+        public const string DotNet60SdkVersion = "6.0.201";
         public const string DotNet60RtmSdkVersion = "6-0-100-rtm-21527-11";
```

Comment:
Do we still need this `DotNet60RtmSdkVersion`

Reply:
I can double check. And remove in https://github.com/microsoft/Oryx/pull/1314 if not